### PR TITLE
Add descriptions for more subprojects

### PR DESCRIPTION
**Pull request by big-guy**

### CODE REVIEW

File: subprojects/build-events/build.gradle.kts [hunk 1]
```diff
@@ -2,6 +2,8 @@ plugins {
     id("gradlebuild.distribution.api-java")
 }
 
+description = "Implementation of build event services and build event types (work item, tasks, tests, configuration, etc)"
```
Comment (lptr) [inline subprojects/build-events/build.gradle.kts:5]: How does this relate to `:build-operations`?

File: subprojects/model-core/build.gradle.kts [hunk 2]
```diff
@@ -2,6 +2,8 @@ plugins {
     id("gradlebuild.distribution.api-java")
 }
 
+description = "Implementation configuration model types (Providers, software model, conventions)"
```
Comment (lptr) [inline subprojects/model-core/build.gradle.kts:5]:
```suggestion
description = "Implementation of configuration model types and annotation metadata handling (Providers, software model, conventions)"
```

File: subprojects/reporting/build.gradle.kts [hunk 3]
```diff
@@ -4,6 +4,8 @@ plugins {
     id("gradlebuild.distribution.api-java")
 }
 
+description = "Report type classes and plugins for reporting (build dashboard, report container)"
```
Comment (lptr) [inline subprojects/reporting/build.gradle.kts:7]: TIL about build dashboards, hah!

File: subprojects/resources-gcs/build.gradle.kts [hunk 4]
```diff
@@ -2,6 +2,8 @@ plugins {
     id("gradlebuild.distribution.api-java")
 }
 
+description = "Implementation for interacting with GCS repositories"
```
Comment (lptr) [inline subprojects/resources-gcs/build.gradle.kts:5]: What is GCS in this context?

File: subprojects/workers/build.gradle.kts [hunk 5]
```diff
@@ -2,6 +2,8 @@ plugins {
     id("gradlebuild.distribution.api-java")
 }
 
+description = "Infrastructure for running worker processes"
```
Comment (lptr) [inline subprojects/workers/build.gradle.kts:5]: Do I understand correctly? `:worker-processes` is about everything that happens to start a worker process, while `:workers` is code that runs in those processes. Is this correct?

**Review by lptr — COMMENTED** [review]
Thanks for finishing up @big-guy. I left a few comments/suggestions.

File: subprojects/resources-gcs/build.gradle.kts (hunk 4 above)
Comment (ljacomet) [reply subprojects/resources-gcs/build.gradle.kts:5]: Google Cloud Storage

**Review by ljacomet — APPROVED** [review]
Thanks a lot for doing this!

File: subprojects/workers/build.gradle.kts (hunk 5 above)
Comment (big-guy) [reply subprojects/workers/build.gradle.kts]: `:worker-processes` is everything that runs in the worker process to load other things the worker may do. This code runs inside the worker. This is the minimum code to get a classloader stood up and communicate with the build process.

`:workers` is the code to start the process. This code runs inside the build process.

**Comment by big-guy** [comment]
@bot-gradle test and merge

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Pull request merged**